sniffles/ruletrafficgenerator.py

```diff
@@ -1586,11 +1586,15 @@ def generate_full_eval(self, rule=None):
                     path = []
                     self.follow_all_branches(nfa, nfa.start, path)
 
-    def generate_nfa_data(self, rule=None):
+    def generate_nfa_data(self, rule=None, length=0):
```

Comment:
Wouldn't `-1` (random length) be more reasonable default value? It's more likely that "no argument" in calling "generate_nfa_data" means "Just generate data; I don't care how many." than "Do not generate data".